sdk/test/ProtocolTests/Generated/QueryProtocol/QueryLists.cs

```diff
@@ -0,0 +1,232 @@
+                    },
+                    new GreetingStruct{
+                        Hi = "hola",
+                    },
+                },
+            };
+            var config = new AmazonQueryProtocolConfig{
+              ServiceURL = "https://test.com/"
+            };
+
+            var marshaller = new QueryListsRequestMarshaller();
+            var marshalledRequest = ProtocolTestUtils.RunMockRequest(request,marshaller,config);
+            var expectedParams = QueryTestUtils.ConvertBodyToParameters("Action=QueryLists&Version=2020-01-08&ListArg.member.1=foo&ListArg.member.2=bar&ListArg.member.3=baz&ComplexListArg.member.1.hi=hello&ComplexListArg.member.2.hi=hola");
+            foreach(var queryParam in expectedParams.Keys){
+               Assert.IsTrue(marshalledRequest.Parameters.Keys.Contains(queryParam));
+               Assert.AreEqual(WebUtility.UrlDecode(expectedParams[queryParam].ToString()),WebUtility.UrlDecode(marshalledRequest.Parameters[queryParam].ToString()));
+            }
+
+            Assert.AreEqual("POST", marshalledRequest.HttpMethod);
+            Uri actualUri = AmazonServiceClient.ComposeUrl(marshalledRequest);
+            Assert.AreEqual("/", actualUri.AbsolutePath);
+            Assert.AreEqual("application/x-www-form-urlencoded; charset=utf-8",marshalledRequest.Headers["Content-Type"]);
+        }
+        [TestMethod]
+        [TestCategory("ProtocolTest")]
+        [TestCategory("RequestTest")]
+        [TestCategory("AwsQuery")]
+        public void EmptyQueryListsRequest()
+        {
+            var request = new QueryListsRequest{
+                ListArg =  new List<string>()
+                {
+                },
+            };
+            var config = new AmazonQueryProtocolConfig{
+              ServiceURL = "https://test.com/"
+            };
+
+            var marshaller = new QueryListsRequestMarshaller();
+            var marshalledRequest = ProtocolTestUtils.RunMockRequest(request,marshaller,config);
+            var expectedParams = QueryTestUtils.ConvertBodyToParameters("Action=QueryLists&Version=2020-01-08&ListArg=");
+            foreach(var queryParam in expectedParams.Keys){
+               Assert.IsTrue(marshalledRequest.Parameters.Keys.Contains(queryParam));
+               Assert.AreEqual(WebUtility.UrlDecode(expectedParams[queryParam].ToString()),WebUtility.UrlDecode(marshalledRequest.Parameters[queryParam].ToString()));
+            }
+
+            Assert.AreEqual("POST", marshalledRequest.HttpMethod);
+            Uri actualUri = AmazonServiceClient.ComposeUrl(marshalledRequest);
+            Assert.AreEqual("/", actualUri.AbsolutePath);
+            Assert.AreEqual("application/x-www-form-urlencoded; charset=utf-8",marshalledRequest.Headers["Content-Type"]);
+        }
+        [TestMethod]
+        [TestCategory("ProtocolTest")]
+        [TestCategory("RequestTest")]
+        [TestCategory("AwsQuery")]
+        public void FlattenedQueryListsRequest()
```

Comment:
Assuming that this corresponds to https://github.com/smithy-lang/smithy/blob/0284c86dcb910034d3be84c3fcd6b201790cbe81/smithy-aws-protocol-tests/model/awsQuery/input-lists.smithy#L60, can you use the modeled `documentation` as an XML comment to make it clear to a reader what the test is exercising?